api-docs/rst/dev-guide/concepts.rst

```diff
@@ -798,3 +808,36 @@ To get started with RackConnect v3 cloud bursting:
 
 .. _RackConnect product information: http://www.rackspace.com/cloud/hybrid/rackconnect
 .. _Knowledge Center articles for RackConnect: http://www.rackspace.com/knowledge_center/product-page/rackconnect
+
+
+.. _concergence-concept:
+
+Convergence
+~~~~~~~~~~~
+
+Convergence is completely rewritten orchestration layer of autoscale that provides
+higher reliability for scaling up and down by optimizing use of Nova API with retries
+as necessary. In addition, Convergence continuously keeps track of all the servers
+in autoscaling groups, notices when a server is deleted out-of-band or goes into
+an non-ACTIVE state (eg ERROR) and automatically replaces it.
+We call this capability “self-healing”.
+
+A new endpoint :ref:`**converge** <post-create-scaling-group-v1.1-tenantid-converge>`
+is added which triggers convergence on a group. This can be useful when a group's
+status is ERROR which happens when group has invalid launch configuration
+(ex imageRef of deleted image). After correcting the launch configuration, one
+needs to invoke this endpoint to converge the group.
+
```

Comment:
When Autoscale detects either of these issues, it adds a  new :ref:`**converge endpoint** <post-create-scaling-group-v1.1-tenantid-converge>` that triggers convergence on a group.  This feature can be useful if the 
scaling group is in an error state, which happens when a group has an invalid launch configuration, for example a 
configuration that includes an image reference for a deleted image.  After correcting the launch configuration, you can submit a request to the convergence endpoint to converge the group.